src/containers/pages/PlanAssignment/helpers/JurisdictionTableView/helpers/utils.tsx

```diff
@@ -0,0 +1,51 @@
+import { ASSIGN_PLANS, HOME } from '../../../../../../configs/lang';
+import { ASSIGN_PLAN_URL, HOME_URL } from '../../../../../../constants';
+import { TreeNode } from '../../../../../../store/ducks/opensrp/hierarchies/types';
+
+/**
+ * Util function to build out pages for header breadcrumb component
+ * @param props - props required to build out pages
+ */
+export const pagesBuilder = (props: any) => {
```

Comment:
Do we have to use any here

Reply:
addressed via https://github.com/onaio/reveal-frontend/pull/1194/commits/4fe94d99db40c4867eb98b468f296f8244b5f286